test/nbrowser/SelectionSummary.ts

```diff
@@ -132,7 +132,7 @@ describe('SelectionSummary', function () {
     });
     await selectAndAssert({col: 2, row: 0}, {col: 3, row: 5}, {
       dimensions: '6⨯2',
-      count: 11,
+      count: 9,
```

Comment:
Just this test is ok.

I would suggest splitting up this `it` block into 2, just so that this assertion gets its own description. Something like this:

```
  it("displays count if the selection doesn't contain numbers", async function () {
    // Everything but the assertion below.
  });

  it("doesn't count falses", async function () {
    await selectAndAssert({col: 2, row: 0}, {col: 3, row: 5}, {
      dimensions: '6⨯2',
      count: 9,
      sum: null,
    });
  });
```

Reply:
Done ✅